core/debug/debug.c

```diff
@@ -4,7 +4,7 @@
 
 volatile bool in_debugger = false;
 
-CEMU_INLINE uint8_t debug_port_read_byte(const uint32_t addr) {
```

Comment:
It should be removed. There's no point to making that function inline.

On Mon, Jan 4, 2016 at 1:11 PM, Nathaniel van Diepen <
notifications@github.com> wrote:

> In core/debug/debug.c
> <https://github.com/MateoConLechuga/CEmu/pull/2#discussion_r48781614>:
>
> > @@ -4,7 +4,7 @@
> >
> >  volatile bool in_debugger = false;
> >
> > -CEMU_INLINE uint8_t debug_port_read_byte(const uint32_t addr) {
>
> Sounds good to me.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MateoConLechuga/CEmu/pull/2/files#r48781614>.
>